appveyor.yml

```diff
@@ -1,5 +1,7 @@
 os: unstable
 
+image: Visual Studio 2017
```

Comment:
First release note: https://github.com/electron-userland/electron-builder/releases/tag/v19.2.2 
I just didn't question it too hard :)